doc/conf.py

```diff
@@ -39,6 +39,9 @@
     "obj_array.ObjectArray1D": "obj:pytools.obj_array.ObjectArray1D",
     # sympy
     "sp.Matrix": "class:sympy.matrices.dense.DenseMatrix",
+    "sym.Expr": "class:sympy.core.expr.Expr",
+    "sym.Symbol": "class:sympy.core.symbol.Symbol",
+    "sym.Matrix": "class:sympy.matrices.dense.DenseMatrix",
```

Comment:
Hm, it just occurred to me that `sym.Matrix` is not an `sp.Expr` or an `sp.Basic`, but I don't think we want to handle those in `pymbolic`, right?